misc/protobuf-ballerina/src/test/java/org/ballerinalang/protobuf/utils/BTestUtils.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.ballerinalang.protobuf.utils;
+
+import org.ballerinalang.compiler.CompilerPhase;
+import org.ballerinalang.launcher.LauncherUtils;
+import org.ballerinalang.launcher.util.CompileResult;
+import org.ballerinalang.util.diagnostic.DiagnosticListener;
+import org.wso2.ballerinalang.compiler.Compiler;
+import org.wso2.ballerinalang.compiler.util.CompilerContext;
+import org.wso2.ballerinalang.compiler.util.CompilerOptions;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static org.ballerinalang.compiler.CompilerOptionName.COMPILER_PHASE;
+import static org.ballerinalang.compiler.CompilerOptionName.PRESERVE_WHITESPACE;
+import static org.ballerinalang.compiler.CompilerOptionName.SOURCE_ROOT;
+
+/**
+ * Utility methods for unit tests.
+ *
+ * @since 0.94
+ */
+public class BTestUtils {
```

Comment:
What is the reason for adding this class. Can't we reuse the existing class what we have used in unit tests?